config/squareone.yml

```diff
@@ -34,6 +34,10 @@ local-docker:
     - 'build-process.php'
     - 'squareone.yml'
   compose-override: 'docker-compose.override.yml'
+  # path to the composer cache directory (relative to the docker directory)
+  composer-cache: 'composer-cache'
```

Comment:
@jbrinley this was breaking in my tests, and this updated path matches  the one we have in our .gitignores in local projects:

- dev/docker/composer-cache > /application/www/composer-cache
- dev/docker/composer > ~/.composer (contains auth.json, .pub files etc...)

Reply:
That moved in https://github.com/moderntribe/square-one/commit/6416857f3e7fb7fca67696965f857d3343bb9344#diff-66788b0d4a77d4154d4e0981949e95e1

Reply:
Older projects will need a `squareone.yml` pointing to the older directory. But I think `so` should stay in sync with SquareOne.